lib/routes/dataguidance/index.ts

```diff
@@ -0,0 +1,65 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+
+export const route: Route = {
+    name: 'Data Guidance News',
+    example: '/',
+    path: '/news',
+    radar: [
+        {
+            source: ['dataguidance.com/news'],
```

Comment:
This RSS extracts data from `https://www.dataguidance.com/search/news` which should be the same as source
```suggestion
            source: ['dataguidance.com/search/news'],
```